.buildkite/scripts/lifecycle/comment_on_pr.ts

```diff
@@ -33,15 +33,23 @@ const ALLOWED_ENV_VARS = [
   'GITHUB_PR_USER',
 ];
 
+interface CommentOnPRArgs {
```

Comment:
Should the changes in this file be done in a standalone PR?

Reply:
Yea... you're right it would make sense to extract this into it's own PR! I'd do that

Reply:
Created a PR here https://github.com/elastic/kibana/pull/219016